frontend/tests-examples/demo-todo-app.spec.ts

```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
```

Comment:
This is autogenerated example. You can check and decided either you want to introduce playwrigh into the repo or no. After that, let remove it from pull request.

Reply:
Sorry for not being able to check it out earlier. I marked it as `keep-open` and probaly find time to have a look at it next week.